recipes/soprano/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "Soprano" %}
+{% set version = "0.8.13b" %}
```

Comment:
If the 'b' is for beta, it would be good to bump to the full 0.8.13 release.

Reply:
No, sorry, it's a bit of a hack because the only difference between this and `0.8.13` is in the tests - some fixes to make them work better with `pytest`. If you think it's a problem I can just bump to `.14`. I didn't do it because the changes don't affect the functionality of the package itself, this was just to set a specific tag on GitHub.

Reply:
Ok. For future reference, I think the usual way to do this in Python land would be 0.8.13.post1. Regardless, we just don't generally want to be putting pre-releases on the main channel is all, so you're fine.